src/crab/ebpf_domain.hpp

```diff
@@ -1136,14 +1169,8 @@ class ebpf_domain_t final {
         inv.assign(r1.offset, 0);
         inv.assign(r1.type, T_CTX);
 
-        inv += 0 <= variable_t::packet_size();
-        inv += variable_t::packet_size() < MAX_PACKET_OFF;
-        if (global_program_info.type.context_descriptor->meta >= 0) {
-            inv += variable_t::meta_offset() <= 0;
-            inv += variable_t::meta_offset() >= -4098;
-        } else {
-            inv.assign(variable_t::meta_offset(), 0);
-        }
+        initalize_packet(inv);
```

Comment:
Looks like a typo in the function name, I think this should be "initialize_packet"

Reply:
Thanks. I fixed it directly on master.